readme.txt

```diff
@@ -19,7 +19,7 @@ This plugin makes it easy to sell tickets, collect registrations, and manage att
 
 Connect to Stripe and take advantage of one of the world’s most popular payment gateways. Our Stripe integration lets you accept credit card payments on your website, along with additional payment methods including AfterPay, ClearPay, AliPay, Giropay, and Klarna.
 
-https://vimeo.com/480516958
+https://evnt.is/1b5e
```

Comment:
I think we need to leave it this way so it automatically embeds. I'm not completely sure, but I'm assuming this won't embed if we use the shortlink.

Reply:
Ah. I wasn't aware that there was an auto embed feature.